internal/application/command_test.go

```diff
@@ -79,7 +79,7 @@ func mockDic() *di.Container {
 	profile := responses.DeviceProfileResponse{
 		BaseResponse: dtoCommon.BaseResponse{},
 		Profile: dtos.DeviceProfile{
-			Name: "test-profile",
+			DeviceProfileBasicInfo: dtos.DeviceProfileBasicInfo{Name: "test-profile"},
```

Comment:
Why is this part of your PR?

Reply:
Unit tests fail without the change.

Reply:
The specific error is "cannot use promoted field DeviceProfileBasicInfo.Name in struct literal of type dtos.DeviceProfile."

Reply:
@cloudxxx8, is this being fix in another PR?

Reply:
Ohh, I see now. It  because you pulled in latest core contracts you are first to see this. Ok. I am fine with it here. Small nit that it is unrelated to your code.